### ISSUE TITLE
 download ngrinder-agent file only 10 kb

### ISSUE THREAD

**Issue by bozheliu** [naver/ngrinder#229]
hi, I am in github clone latest project and deploy it later, in main page download ngrinder-agent file only 10 kb, the operating environment is windows 8、oracle JDK 1.7、jetty-distribution-9.2.21.v20170120

**Comment by songeunwoo** [comment]
@lopiter 
Could you check this?

**Comment by bozheliu** [comment]
I haven't found the cause of the problem yet.

**Comment by lopiter** [comment]
hi @bozheliu ?

i tested like you  but there is nothing problem

could you explain more detail like below??

1. git clone from github page
2. use develop branch
3. run maven package
4. run ngrinder war file in ngrinder-controller target directory ( java -jar ngrinder.war)
5. download agent

* before test , i deleted ngrinder home directory

thank you

**Comment by bozheliu** [comment]
Thank you for your reply. @lopiter 
I am the latest develop branch directly from the github page, and I used the develop branch too. The project is then imported into intellij IDEA,Integrated jetty container in intellij IDEA. After the project is started, download the ngrinder-agent is only 10 kb. I tried to stop the project service and delete ${USER}/. ngrinder and. ngrinder  folder, restart the project again, and the ngrinder, or the same problem as before. However, I use the maven command to package the project and install it under the webapps directory of the web container, start the project, and download the ngrinder - the agent is normal. I see people encounter the same problem on "http://ngrinder.642.n7.nabble.com/nGrinder3-3-Agent-10KB-td1453.html".
 Thank you again very much.
